### ISSUE TITLE
Explain traffic light levels to the user

### ISSUE THREAD

**Issue by Robinlovelace** [cyipt/actdev-ui#54]
I imagine a common question will be "What do the colours mean?"

That could go in a FAQ section post workshop (we will find out what most common questions are then).

Could there also be a 'tooltip' of popup when you click on one of the site ratings that says something along the lines of:

"The colours represent a simple traffic light rating scheme for active travel, from red (poor active travel performance) to green (relatively good active travel performance) based on the areas current and estimated potential future levels of walking, cycling and driving. For active travel values less than 20% are marked as red, values between 20% and 30% are rated as amber, values over 30% are rated as green. The equivalent values for walking and cycling are 0-10%, 10-15% and 15%+ rated as green. For driving the equivalent values are 50%+ (red), 50%-20% (amber) and less than 20% (green). See the project's [issue tracker](https://github.com/cyipt/actdev-ui/issues/51) for details."

Too wordy for a tooltip? Please provide guidance on FAQ page, tooltips or other place to put this and we can things like this documented.

Looking great with the new colour breaks btw, I like the replacement of distance with walking, more consistent.

**Comment by Siequnu** [comment]
Pretty wordy, I'm not sure it'd fit on a tooltip! But the idea is good — should we have an image based legend instead?

**Comment by Siequnu** [comment]
I’ve talked action on this and will explore tomorrow morning text and visual approaches to this.

**Comment by Siequnu** [comment]
I've made a little popup that is programatically generated for each stat. It contains a coloured legend. @Robinlovelace could you verify and close, or give further suggestions? If more info is needed, I suggest we include a deeplink to a paragraph in the report/methodology which explains this.

<img width="722" alt="Screenshot 2021-03-10 at 11 40 10" src="https://user-images.githubusercontent.com/11821606/110572968-65070080-8195-11eb-9f5d-2b7314cad846.png">

**Comment by Robinlovelace** [comment]
Looking good but could the text shown say:

```
50%+
20-50%
0-20%
```

So it's clear that these are bin ranges not specific values in a continuous colourscheme.

**Comment by mvl22** [comment]
![Screenshot 2021-03-10 at 20 02 56](https://user-images.githubusercontent.com/361423/110690283-a9b58a80-81db-11eb-886a-5e56e95d993c.png)

**Comment by Robinlovelace** [comment]
Thanks Martin, looking great!
